### PR TITLE
Assign demo users to default Program #641

### PR DESCRIPTION
**Pull request by Menda**
Purpose
=======

All Demo Users should have Workflow Teams access to the default WFL1:
"Humanitarian Response to the Syrian Crisis".

Ticket
=====

https://github.com/toladata/TolaDataV2/issues/641

### CODE REVIEW

File: workflow/signals.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    except WorkflowLevel1.DoesNotExist:
+        logger.warning(
+            'Working on branch %s but the default program "#%d: %s" does not '
+            'exist. Maybe initial data was not loaded with loadinitialdata?',
+        )
+        return
+    else:
+        role = Group.objects.get(name=ROLE_VIEW_ONLY)
+        WorkflowTeam.objects.create(workflow_user=instance, role=role,
+                                    workflowlevel1=wflvl1)
```
Comment (jefmoura) [inline workflow/signals.py]: You're using the `else`, so you don't need the `return` in the `except` clause.

File: workflow/signals.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+from tola.management.commands.loadinitialdata import (
+    DEFAULT_WORKFLOWLEVEL1_ID, DEFAULT_WORKFLOWLEVEL1_NAME)
+from workflow.models import (TolaUser, WorkflowLevel1, WorkflowTeam,
+                             ROLE_VIEW_ONLY)
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(signals.post_save, sender=TolaUser)
+def add_users_to_default_wflvl1(sender, instance, **kwargs):
```
Comment (jefmoura) [inline workflow/signals.py]: If the TolaUser is being updated instead, you'll create a new WorkflowTeam with the default program, even though the user already has it. You should check this case as well.

Comment (Menda) [reply workflow/signals.py]: Good eye! I didn't think about this possibility

**Comment by Menda** [comment]
@jefmoura done

**Pull request merged**